### ISSUE TITLE
Bug in swin backbone

### ISSUE THREAD

**Issue by Jaykob** [open-mmlab/mmpose#1993]
**Describe the bug**

When trying to train a swin net, for example with `configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_swin-b-p4-w7_8xb32-210e_coco-256x192.py`, it fails because in line 680 https://github.com/open-mmlab/mmpose/blob/3b0b41bdd9710e17e8ea29e6e0e0b8b6322683b7/mmpose/models/backbones/swin.py#L680
 you try to access the first element of an empty OrderedDict (just created one line above). Did you mean to access `_state_dict` instead of `state_dict`?

**Reproduction**

- What command or script did you run?

```
 python tools/train.py configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_swin-b-p4-w7_8xb32-210e_coco-256x192.py
```

- What config did you run?

```
configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_swin-b-p4-w7_8xb32-210e_coco-384x288.py
```

**Comment by ly015** [comment]
Thanks very much for reporting this bug. We will fix it asap.

**Comment by ly015** [comment]
This bug will be fixed in #1995 in dev-1.x branch first. And it will be merged into the mainstream 1.x branch in the next release, which should be soon.

**Comment by Jaykob** [comment]
Nice, thanks for the very quick fix 🙂
